corehq/apps/app_manager/templates/app_manager/form_summary_base.html

```diff
@@ -177,6 +177,9 @@ <h5>
         <span data-bind="css: $parent.getLoadSaveDiffClass('save_properties', $data.case_type, $data.property), popover: $parent.getLoadSavePopover('save_properties', $data.case_type, $data.property)">
           <span class="text-muted" data-bind="text:$data.case_type"></span> -
           <!-- ko text: $data.property --><!-- /ko -->
+          <!-- ko if: $data.update_mode === 'edit' -->
+          <sup class="text-muted">{% trans 'save only on edit' %}</sup>
```

Comment:
I wish there was an icon that clearly implied "save only on edit". In lieu of that, I like this UI treatment.